src/native/chunk/splitter.cpp

```diff
@@ -44,8 +44,21 @@ Splitter::Splitter(
     if (_calc_md5) {
         extern bool fips_mode;
         if (fips_mode) {
-            posix_memalign((void**)&_md5_mb_mgr, 16, sizeof(MD5_HASH_CTX_MGR));
-            posix_memalign((void**)&_md5_mb_ctx, 16, sizeof(MD5_HASH_CTX));
+            int result;
+            result = posix_memalign((void**)&_md5_mb_mgr, 16, sizeof(MD5_HASH_CTX_MGR));
+            if (result == 0) {
+                printf("Memory allocation successful. Aligned address: %p\n", _md5_mb_mgr);
+                free(ptr);
```

Comment:
why `free`?

Reply:
Actually, its just a testing code to fix the CI job. I forgot to remove it.